schemas/rdf/examples/KapitalVerwaltungsschaleExample.ttl

```diff
@@ -8,24 +8,23 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix vann: <http://purl.org/vocab/vann/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@base <http://admin-shell.io/aas/3/0/RC01/> .
+@base <http://admin-shell.io/aas/3/0/RC02/> .
 
 
 <http://customer.com/assets/MaxRotationSpeed>
-    a <https://admin-shell.io/aas/3/0/RC01/Property> ;
-    <https://admin-shell.io/aas/3/0/RC01/Property/valueType> "xsd:integer" ;
-    <https://admin-shell.io/aas/3/0/RC01/Property/value> "5000"^^xsd:integer ;
-    <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "MaxRotationSpeed"^^xsd:string ;
-    <https://admin-shell.io/aas/3/0/RC01/Referable/category> "PARAMETER"^^xsd:string ;
-    <https://admin-shell.io/aas/3/0/RC01/HasKind/kind> <https://admin-shell.io/aas/3/0/RC01/ModelingKind/Instance>  ;
-    <https://admin-shell.io/aas/3/0/RC01/HasSemantics/semanticId> [
-       a <https://admin-shell.io/aas/3/0/RC01/Reference> ;
-       <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "AReference"^^xsd:string ;
-       <https://admin-shell.io/aas/3/0/RC01/Reference/key> [
-         a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-         <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/ConceptDescription> ;
-         <https://admin-shell.io/aas/3/0/RC01/Key/value> "0173-1#02-BAA120#008"^^xsd:string ;
-         <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Irdi> ;
+    a <https://admin-shell.io/aas/3/0/RC02/Property> ;
+    <https://admin-shell.io/aas/3/0/RC02/Property/valueType> "xsd:integer" ;
+    <https://admin-shell.io/aas/3/0/RC02/Property/value> "5000"^^xsd:integer ;
+    <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "MaxRotationSpeed"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/Referable/category> "PARAMETER"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/HasKind/kind> <https://admin-shell.io/aas/3/0/RC02/ModelingKind/Instance>  ;
+    <https://admin-shell.io/aas/3/0/RC02/HasSemantics/semanticId> [
+       a <https://admin-shell.io/aas/3/0/RC02/ModelReference> ;
+       <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "AReference"^^xsd:string ;
```

Comment:
Why does a semanticId have an idShort (it is not a Referable)?

Reply:
Deleted.